sql-statements/sql-statement-load-data.md

```diff
@@ -12,107 +12,157 @@ The `LOAD DATA` statement batch loads data into a TiDB table.
+    ('FORMAT' ('DELIMITED DATA' | 'SQL FILE' | 'PARQUET'))?
+
+Fields ::=
+    ('TERMINATED' 'BY' stringLit
+    | ('OPTIONALLY')? 'ENCLOSED' 'BY' stringLit
+    | 'ESCAPED' 'BY' stringLit
+    | 'DEFINED' 'NULL' 'BY' stringLit ('OPTIONALLY' 'ENCLOSED')?)?
+
+LoadDataOptionListOpt ::=
+    ('WITH' (LoadDataOption (',' LoadDataOption)*))?
+
+LoadDataOption ::=
+    detached | batch_size '=' numberLiteral
 ```
 
 ## Parameters
 
-### `LocalOpt`
+### `LOCAL`
+
+You can use `LOCAL` to specify to import data files located on the client, where the file parameter must be the file system path on the client.
+
+### S3 storage
+
+If you do not specify `LOCAL`, the file parameter must be a valid S3 path, as detailed in [external storage](/br/backup-and-restore-storages.md).
 
-You can specify that the imported data file is located on the client or on the server by configuring the `LocalOpt` parameter. Currently, TiDB only supports data import from the client. Therefore, when importing data, set the value of `LocalOpt` to `Local`.
+When the data files are stored on S3, you can import individual files or use the wildcard character `*` to match multiple files to be imported. Note that wildcards do not recursively process files in subdirectories. The following are some examples:
 
-### `Fields` and `Lines`
+- Import a single file: `s3://<bucket-name>/path/to/data/foo.csv`
+- Import all files in the specified path: `s3://<bucket-name>/path/to/data/*`
+- Import all files ending with `.csv` under the specified path: `s3://<bucket-name>/path/to/data/*.csv`
+- Import all files prefixed with `foo` under the specified path: `s3://<bucket-name>/path/to/data/foo*`
+- Import all files prefixed with `foo` and ending with `.csv` under the specified path: `s3://<bucket-name>/path/to/data/foo*.csv`
 
-You can specify how to process the data format by configuring the `Fields` and `Lines` parameters.
+### `FORMAT`
 
-- `FIELDS TERMINATED BY`: Specifies the separating character of each data.
-- `FIELDS ENCLOSED BY`: Specifies the enclosing character of each data.
-- `LINES TERMINATED BY`: Specifies the line terminator, if you want to end a line with a certain character.
+You can use the `FORMAT` parameter to specify the format of the data file. If you do not specify this parameter, you can use the format defined by `DELIMITED DATA`, which is the data format supported by MySQL `LOAD DATA`.
+
+### `Fields`, `Lines`, and `Ignore Lines`
+
+You can specify the `Fields`, `Lines`, and `Ignore Lines` parameters only when the data format is `DELIMITED DATA`.
+
+You can use the `Fields` and `Lines` parameters to specify how to handle the data format.
+
+- `FIELDS TERMINATED BY`: specifies the data delimiter.
+- `FIELDS ENCLOSED BY`: specifies the enclosing character of the data.
+- `LINES TERMINATED BY`: specifies the line terminator, if you want to end a line with a certain character.
+
+You can use `DEFINED NULL BY` to specify how NULL values are represented in the data file.
```

Comment:
We should give examples on this new syntax, will do it later

Reply:
added as per cn version